sections/reviews.md

```diff
@@ -0,0 +1,74 @@
+
+## How
+
+The process for code reviews is straight forward as long as you are already following our coding standards for working with version control, and in particular our git flow-influenced process of branch management. The bottom line is that, before any code hits `master`, it must be reviewed by a colleague.
+
+### Submitting code for review
+
+Submitting code for review should follow these steps:
+
+1. Work out of your own fork of the main repository (usually `origin`).
+2. Always, always work from a feature or fix branch that is checked out of `master`.
+3. Each feature or fix branch should be focused on a discrete unit of work.
+4. When your unit of work is complete, submit a pull request against the `master` branch on `origin`.
+5. Wait for the CI server to run, validating your tests pass on all target environments. If you do not have tests, or have not configured CI, do that immediately before proceeding.
+6. If CI is green, ask a colleague to review your pull request (we will automate this in the near future). This preferably will be a colleague who is not working on your project.
+7. Address any questions from the code review. Sometimes this will involving refactoring, other times it will just mean answering questions.
+8. When the reviewer indicates that the pull request is ready to merge, you may merge into `master`.
+
+### Reviewing code in a pull request
+
+Reviewing someone else's code should follow the following steps:
+
+1. Respond to a request for a code review immediately. Indicate when you can do it ("now", "in the next hour", and so on).
+2. Review the pull request, using inline commenting for specific issues.
+3. If you are satisfied with the pull request (without any questions, or after your questions have been answered), then explicitly state that the pull request is ready to be merged as a comment on the pull request.
+
+### A simple checklist for reviewers
+
+Remember that the primary reason for doing code reviews at Open Knowledge International is to distribute knowledge across our team. This does mean identifying areas the code could be improved and asking useful questions, but it does not mean you go too deeply on issues that are clearly subjective.
+
+Here are some points to hold in mind and guide the review process:
+
+* What is the context for the code in this pull request (what does the app/library actually do)?
+* What is the desired functionality being introduced or fixed?
+* Is the code idiomatic for the language and/or framework it is written in?
+
+There are some clear red flags that, on seeing any of them, you should immediately ask your colleague to address before continuing with any code review:
+
+* There is no CI
+* There are no tests
+* There are tests that fail
+* There is code that is not linted
+
+### Prioritizing code reviews
```

Comment:
@akariv understood, but I don't want to move to a pull model now. Perhaps once we have integrated code reviews into our daily work practice fully.